tools/clang/test/HLSL/workgraph/dispatchgrid_diags.hlsl

```diff
@@ -153,6 +153,7 @@ void node20(DispatchNodeInputRecord<C> input)
 { }
 
 struct D {
+  // expected-error@+1 {{SV_DispatchGrid should be 32/16 bit uint scalar or vector/array up to 3 elements}}
   uint4 grid1 : SV_DispatchGrid;   // expected-note {{other SV_DispatchGrid defined here}}
```

Comment:
Since this isn't what this was meant to test, I'm inclined to change the type to uint3 instead of adding another expected diagnostic as the test you've added does that quite well.